src/client-config.js

```diff
@@ -4,11 +4,22 @@
  * When webpack builds the client-side code it exposes
  * the clientConfig config via the definePlugin as CLIENT_CONFIG.
  */
+import { oneLine } from 'common-tags';
 
-const clientConfig = new Map();
+export class ClientConfig {
+  constructor(objData) {
+    Object.assign(this, {
```

Comment:
There's a test which should make it obvious if this gets changed. Although I guess someone might rename `objData` while they're at it and the test would still pass. Maybe the test should check `Object.keys(new ClientConfig({})) === ['get', 'has']`?